server/service/src/sync/translations/stock_line.rs

```diff
@@ -96,10 +99,25 @@ impl SyncTranslation for StockLineTranslation {
             supplier_id,
             barcode_id,
             item_variant_id,
+            vaccine_doses,
         } = serde_json::from_str::<LegacyStockLineRow>(&sync_record.data)?;
 
         let barcode_id = clear_invalid_barcode_id(connection, barcode_id)?;
         let location_id = clear_invalid_location_id(connection, location_ID)?;
+
+        let item_doses = ItemRowRepository::new(connection)
```

Comment:
Also a new column in OG, so for existing records, saving item.doses for vaccine items... maybe OG should be doing this?

Reply:
Hmm, OG central can't edit lines belonging to remotes

Reply:
> Also a new column in OG

In that case it should be om_doses. And actually iI recommend we add a new field called om_data, we had a chat with @jmbrunskill about this, it would allow easier extension and differentiation between 'nulls' and 0s.

This om_data is a json filed.

I think we should only do this check if om_data is not set

The more I think of use case the more I think we should just use the item or item variant dose value for all of this.

i'll get back to you tomorrow

Reply:
Yup, only saving if stock_line.doses isn't saved. Will rename or swap to json file if you want

Okiedok